src/app/app-routing.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AppComponent } from './app.component';
+import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+
+const routes: Routes = [
+  //{path:'',component: AppComponent,pathMatch: 'full'},
+  {path: '' , redirectTo: 'AppComponent', pathMatch: 'full'},
+  {path:':id/itineraries', component: ItinerarylistComponent,pathMatch: 'full'},
+  { path: '**', redirectTo: '', pathMatch: 'full' }
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
  { path: "**", redirectTo: '', pathMatch: 'full' }
```